src/openrpc/lifecycle_management.json

```diff
@@ -85,7 +85,25 @@
             "result": {
                 "name": "result",
                 "schema": {
-                    "$ref": "#/components/schemas/AppSession"
+                    "oneOf": [
+                        {
+                            "$ref": "#/components/schemas/AppSession"
+                        },
+                        {
+                            "type": "object",
+                            "properties": {
+                                "state": {
+                                    "const": "pending"
```

Comment:
Is it required to define "state" in the schema with all possible const values (pending/created/anyother_state) ?

Reply:
There are two possible values, pending and created. Those are the two consts in the oneOf.
If state is pending, that means sessionId, loadedSessionId and activeSessionId must all be undefined.
If state is created then it follows the spec of AppSession, which says that appId, sessionId and loadedSessionId must be defined.